src/logic/safe/transactions/awaitingTransactions.ts

```diff
@@ -6,6 +6,10 @@ export const getAwaitingTransactions = (allTransactions, cancellationTransaction
   }
 
   const allAwaitingTransactions = allTransactions.map((safeTransactions) => {
+    if (!safeTransactions) {
```

Comment:
This is rather fixing the `consequence` than fixing the root of the problem. Could you please try to find the root of the problem? Why is safeTransactions undefined? It never should be, it should be an empty list